scripts/linz-bde-schema-load.bash

```diff
@@ -98,7 +98,7 @@ fi
 if test $DBPATCH_SUPPORTS_STDOUT != yes; then
     echo "WARNING: dbpatch-loader does not support stdout mode, working in non-transactional mode" >&2
     echo "HINT: install dbpatch 1.4.0 or higher to fix this." >&2
-    ${DBPATCH_LOADER} ${DBPATCH_OPTS} ${PGDATABASE} _patches || exit 1
+    ${DBPATCH_LOADER} ${DBPATCH_OPTS} "${PGDATABASE}" _patches || exit 1
```

Comment:
All of these variables should be quoted. ShellCheck unfortunately doesn't suggest quoting variables which are currently set to a static string with no special characters.